tailwind.config.js

```diff
@@ -1,6 +1,7 @@
 const colors = require('tailwindcss/colors')
 
 module.exports = {
+  mode: 'jit',
```

Comment:
What benefits does the `jit` mode provide?

Reply:
tailwind v2.1+ no longer needs extra tailwindcss-jit package, only need to set the mode to 'jit' here.